scripts/gulp/tasks/bundle.ts

```diff
@@ -14,16 +14,18 @@ const packageName = config.package;
 // Clean
 // ----------------------------------------
 
-task('bundle:package:clean', () =>
-  del(
-    [
-      `${paths.packageDist(packageName)}/es/*`,
-      `${paths.packageDist(packageName)}/commonjs/*`,
-      `${paths.packageDist(packageName)}/dts`,
-    ],
+task('bundle:package:clean', () => {
+  const pathsToDelete = [
+    `${paths.packageDist(packageName)}/es/*`,
+    `${paths.packageDist(packageName)}/commonjs/*`,
+    `${paths.packageDist(packageName)}/dts`,
```

Comment:
You can probably get rid of the windows fix below by doing this:
```suggestion
    paths.packageDist(packageName, 'es', '*'),
    paths.packageDist(packageName, 'es', '*'),
    paths.packageDist(packageName, 'es', '*'),
```